src/multimeditron/model/modalities/image_modality.py

```diff
@@ -16,6 +16,7 @@ class ImageConfig(BaseModalityConfig):
         max_batch_size (int): Maximum batch size for processing.
         clip_name (str): Name of the CLIP model to use as the feature extractor.
         projection_type (str): Type of projection network (e.g., "mlp").
+        use_2d_adapation (bool): Whether to use the 2D positional embeddings adaptation for 1D llm without retraining.
```

Comment:
I'm not sure if we should put this in the configuration file as this would be redundant with the parameter of the DataCollator. Maybe we can move this as an optional argument of the process function (which is called indirectly by the collator)?